lldb/source/Target/RegisterContextUnwind.cpp

```diff
@@ -2042,7 +2043,7 @@ bool RegisterContextUnwind::ReadFrameAddress(
       return true;
     }
     UnwindLogMsg("Failed to set CFA value via DWARF expression: %s",
-                 error.AsCString());
+                 llvm::toString(result.takeError()).c_str());
```

Comment:
Does `takeError` disarm the result? I am wondering if this will assert that no one has handled the error here.

Reply:
Yup